sdk/storage/azure-storage-file-datalake/src/test/java/com/azure/storage/file/datalake/APISpec.groovy

```diff
@@ -14,10 +14,7 @@ import com.azure.core.util.Configuration
 import com.azure.core.util.logging.ClientLogger
 import com.azure.identity.EnvironmentCredentialBuilder
 import com.azure.storage.common.StorageSharedKeyCredential
-import com.azure.storage.file.datalake.models.FileSystemItem
-import com.azure.storage.file.datalake.models.LeaseStateType
-import com.azure.storage.file.datalake.models.ListFileSystemsOptions
-import com.azure.storage.file.datalake.models.PathProperties
+import com.azure.storage.file.datalake.models.*
```

Comment:
Checkstyle should be mad at this.

Reply:
I think checkstyle doesnt run on test files

Reply:
I believe Checkstyle is suppressed on `.groovy` files